<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MattoxWorkflowDialog</class>
 <widget class="QDialog" name="MattoxWorkflowDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>828</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Palatino Linotype</family>
    <pointsize>11</pointsize>
    <weight>9</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Mattox Workflow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(25, 49, 91);
font: 75 11pt &quot;Palatino Linotype&quot;;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="4">
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frame_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);
font-weight: bold;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,0,0,0,0,0,0,0,0">
      <property name="verticalSpacing">
       <number>24</number>
      </property>
      <item row="4" column="0">
       <widget class="QListWidget" name="community_control_entries_listWidget"/>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="close_dialog_Button">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(160, 160, 160);
font-weight: bold;</string>
        </property>
        <property name="text">
         <string>Close Workflow</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QListWidget" name="scram_gps_entries_listWidget"/>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="Line" name="line_2">
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;PRETRIAL ENTRIES&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Palatino Linotype</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>COMMUNITY CONTROL ENTRIES</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="load_new_entries_Button">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(160, 160, 160);
font-weight: bold;</string>
        </property>
        <property name="text">
         <string>Check For New Entries</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="open_entry_Button">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 255, 0);
font: 75 11pt &quot;Palatino Linotype&quot;;
font-weight: bold;</string>
        </property>
        <property name="text">
         <string>Open Selected Entry for Review</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QPushButton" name="delete_entry_Button">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 96, 82);
font: 75 11pt &quot;Palatino Linotype&quot;;
font-weight: bold;</string>
        </property>
        <property name="text">
         <string>Delete Entry from Workflow</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>proceed_to_ability_to_pay()</slot>
  <slot>close_window()</slot>
  <slot>add_offense()</slot>
  <slot>amend_offense()</slot>
  <slot>proceed_to_sentencing()</slot>
 </slots>
</ui>
